### PR TITLE
Remove Goerli support

### PR DESCRIPTION
**Pull request by michaelsproul**
## Proposed Changes

Goerli is dead and deprecated. This PR removes Goerli support from Lighthouse. Flags like `--network goerli` and `--network prater` will no longer be accepted.

## Additional Info

The validator manager test vectors have been regenerated for Holesky. This required some portability fixes to the generator script.

### CODE REVIEW

File: testing/web3signer_tests/src/lib.rs [hunk 1]
```diff
@@ -901,13 +901,14 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn prater_base_types() {
-        test_base_types("prater", 4246).await
+    async fn mainnet_bellatrix_types() {
+        test_base_types("mainnet", 4244).await
```
Comment (jimmygchen) [inline testing/web3signer_tests/src/lib.rs]: Looks like we're testing base types here
```suggestion
        test_bellatrix_types("mainnet", 4244).await
```

**Review by jimmygchen — CHANGES_REQUESTED** [review]
Love deleting stuff 🎉  just a small comment above.

File: testing/web3signer_tests/src/lib.rs (hunk 1 above)
Comment (michaelsproul) [reply testing/web3signer_tests/src/lib.rs]: good catch!

**Review by jimmygchen — APPROVED** [review]
LGTM!

**Comment by michaelsproul** [comment]
@mergifyio queue

**Pull request merged**